docs/prysm-usage/grafana-dashboard.md

```diff
@@ -0,0 +1,145 @@
+---
+id: grafana-dashboard
+title: Monitoring and alerting with Grafana
+sidebar_label: Monitoring with Grafana
+---
+
+Now that you have your validator and node process running, you will probably need a nice dashboard and alert system to ensure at maximum the profitability of your staking ETH. Here is a simple guide that explain how to get one, without any developer skill.
+
+Here is the result of what you will get if you follow this guide!
+![Grafana dashboard for prysm node and validator](https://imgur.com/nbI9KPP.png "Grafana dashboard for prysm node and validator")
```

Comment:
Instead of an imgur link, can we upload the image to this repo? That way we don't lose it someday. I'm also not sure if these links can be hijacked where the image is updated to display undesirable information :scream: